classes/module/Module.php

```diff
@@ -417,7 +417,9 @@ public function install()
         $result = Db::getInstance()->insert($this->table, ['name' => $this->name, 'active' => 1, 'version' => $this->version]);
         if (!$result) {
             $this->_errors[] = Context::getContext()->getTranslator()->trans('Technical error: PrestaShop could not install this module.', [], 'Admin.Modules.Notification');
```

Comment:
Yeah, well, I don't think I ever saw this message. Probably this is why we never had an error related to `uninstallTabs` 😬

Reply:
:+1:

Reply:
https://github.com/PrestaShop/PrestaShop/issues/15927 😄